extensions/wikia/Security/classes/CSRFDetector.class.php

```diff
@@ -36,6 +36,18 @@ public static function onRequestWasPosted() {
 	}
 
 	/**
+	 * In some cases we consider GET requests to be secure when they're provided with a secret token
+	 *
+	 * @see PLATFORM-2207
+	 */
+	public static function markRequestAsSecure( $caller ) {
```

Comment:
The name of the method is misleading in my opinion. It should either mark all internal checks as passed or be renamed to describe that it only marks the HTTP method to be accepted